templates/base/storybook/.storybook/preview-head.html

```diff
@@ -0,0 +1,8 @@
+<style>
+  body {
+    padding: 0 10px;
+  }
+  .hide-default-sample {
```

Comment:
`.hide-default-sample {
    display: none !important;
  }`

Don't need this anymore